src/AppBundle/Resources/views/ContextImportExport/importContext.html.twig

```diff
@@ -23,6 +23,17 @@
             {% endif %}
 
             <div class="form-group row">
+                <label class="col-sm-2 form-control-label" for="name">Description:</label>

```

Comment:
Should be 'for="description"' here. But I'll fix that small detail after the merge.